src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -24,12 +24,16 @@ public class CliSyntax {
     public static final SeplendidParameter PARAMETER_PARTNERDESCRIPTION = new SeplendidParameter("partnerdescription");
     public static final SeplendidParameter PARAMETER_UNIVERSITY = new SeplendidParameter("university");
     public static final SeplendidParameter PARAMETER_ATTRIBUTE = new SeplendidParameter("attribute");
+    public static final SeplendidParameter PARAMETER_LOCALATTRIBUTE =
+            new SeplendidParameter("localattribute");
     public static final SeplendidParameter PARAMETER_UNIVERSITYNAME = new SeplendidParameter("university");
     public static final SeplendidParameter PARAMETER_CONTENT = new SeplendidParameter("content");
     public static final SeplendidParameter PARAMETER_TAGS = new SeplendidParameter("tags");
     public static final SeplendidParameter PARAMETER_INFORMATION = new SeplendidParameter("information");
     public static final SeplendidParameter PARAMETER_MAPPINGATTRIBUTE =
             new SeplendidParameter("mappingattribute");
     public static final SeplendidParameter PARAMETER_QUERY = new SeplendidParameter("query");
+    public static final SeplendidParameter PARAMETER_UPDATEDVALUE = new SeplendidParameter("editedValue");
```

Comment:
As mentioned below as well, would it be better to separate PARAMETER_LOCALUPDATEDVALUE and PARAMETER_PARTNERUPDATEDVALUE?